### ISSUE TITLE
Force files to open in a specific opened window

### ISSUE THREAD

**Issue by sborkin** [Microsoft/vscode#53544]
Currently when opening a file it opens in the last selected VS code window, I'd like the option to make all the files open in a window which I choose.

The use case is I have multiple windows open, for separate projects. When I open a file which is not part of one of those projects (for example simple text file which is on the desktop) I want it to open in a window which I create just for those type of "external" files. It allows to easily find them in case I move between windows

Thank you!

**Comment by bpasero** [comment]
I find this a bit weird to be honest, how would you on the command line target the window to open in?

**Comment by sborkin** [comment]
it shouldn't be done manually on the command line, I see it as an automatic logic done internally by vscode:

1. we add a new ui option to vscode `set a unique title per window`
2. we add a new ui option to vscode to `open all files which are not part of any open project in a window with a title...`
3. user sets above option (2nd) as "title_2"
4. user opens `window1` for a `project1`
5. user opens empty `window2`
6. user sets title for `window2` as "title_2"
7. user is currently working in `window1` 
7. user opens a file which is not part of `project1`
8. vscode opens the file in `window2` due to the set value

**Comment by sborkin** [comment]
again the use case is I have 3 open windows in parallel for different projects.

 also I have few files with some common bash commands, stored links, or some other info for example, they are not part of any of the projects

I would be nice that if I open the file with the bash commands it would not open in any of the open windows for the projects but in a separate window which I can always switch to instead of searching it through the open projects

**Comment by bpasero** [comment]
I appreciate the clarification but I do not see us implementing this in the future. I will close this as "out of scope".
